docs/CHANGELOG.asciidoc

```diff
@@ -30,6 +30,223 @@ This section summarizes the changes in each release.
+* [Fixes #40493] [Infra UI] Inventory should display nodes without names {pull}40804[#40804]
+* [Fixes #39991] [Infra and Logs UI] Fixes &quot;sticky filter&quot; problem {pull}40226[#40226]
+* [Infra UI] Add temporary workaround for too small max-width. {pull}39890[#39890]
+* [Fixes #39892] [Infra UI] Remove queries for log-related metadata {pull}40130[#40130]
+* [Fixes #39944] [Logs UI] Fix inaccuracy when jumping to a faraway time target {pull}40303[#40303]
+* [Fixes #39913] [Infra UI] Widen Group By dropdown and add tooltips to custom metrics {pull}40692[#40692]
+Machine Learning::
+* [ML] Data Frame - Disable link to create new data frame job when no permissions {pull}40268[#40268]
+* [ML] Data frame - ensure pivot preview works with comma-separated index patterns  {pull}40282[#40282]
+* [Fixes #40562] [ML] Single Metric View: ensure charts loaded correctly when using scripted fields {pull}40682[#40682]
+* [Fixes #39770] [ML] Fixes results time range if anomalies occur after data stops {pull}40503[#40503]
+* [Fixes #32304] [ML] Fixes positioning of annotations brush after window resize {pull}40780[#40780]
+* [Fixes #33416] [ML] Fixes position of Single Metric zoom when at start of range {pull}40854[#40854]
+* [Fixes #41135] [ML] Fixes model memory limit for metricbeat system module jobs {pull}41747[#41747]
+* [Fixes #38872] [ML] Data Frames: Fixes crash of the source table for complex field values. {pull}39878[#39878]
+* [ML] Data Frames: Fixes transform list paging and sorting. {pull}40009[#40009]
+* [Fixes #38654] [ML] Data Frames: Fix form validation {pull}40427[#40427]
+* [Fixes #38395] [ML] Data Frames: Fixes search triggering onChange on every keystroke. {pull}40607[#40607]
+* [Fixes #40960] [ML] Fix loading forecasts from anomaly job list. {pull}41034[#41034]
+* [ML] Data frames: Fix source table for cells of type object. {pull}41234[#41234]
+Management::
+* [Fixes #40254] Fix aggs autocompletion in Console. {pull}40816[#40816]
+* [Fixes #39087] Remove config from IndexPattern instance {pull}40146[#40146]
+Monitoring::
+* Add log message for error scenario around missing licensing {pull}40660[#40660]
+* [Fixes #36738] Issue 36738: Added zoomout functionality to all charts {pull}39072[#39072]
+* [Fixes #28471] Issue 28471: Added strict id checking {pull}40673[#40673]
+Observability::
+* [Fixes #39679] [Uptime] Fix filter status bar location selection {pull}41382[#41382]
+* [Fixes #40171] [Uptime] [Bug] Handle all supported empty states {pull}40194[#40194]
+* [Uptime] Fix unsigned integer parsing error and add tests {pull}40526[#40526]
+* [Fixes #40582] [Uptime] Change page headlines to use h1 element {pull}40637[#40637]
+* [Fixes #40583] [Uptime] Add explicit alignments to table columns {pull}40680[#40680]
+* [Uptime] Fix duration chart for Safari {pull}41619[#41619]
+Operations::
+* [Fixes #31549] configurable global socket timeouts {pull}31603[#31603]
+Other::
```

Comment:
Except where noted, the issues under Other can be moved to under a category titled "Visualization"